code.json

```diff
@@ -0,0 +1,47 @@
+[
+    {
+      "name": "plio",
+      "organization": "U.S. Geological Survey",
+      "description": "A planetary surface data input/output library written in Python.",
+      "version": "1.5.0",
+      "status": "Release",
+
+      "permissions": {
+        "usageType": "openSource",
+        "licenses": [
+          {
+            "name": "Public Domain, CC0-1.0",
+            "URL": "https://code.usgs.gov/astrogeology/plio/-/blob/master/LICENSE.md"
+          }
+        ]
+      },
+  
+      "homepageURL": "https://plio.readthedocs.io/en/latest/?badge=latest",
+      "downloadURL": "https://github.com/USGS-Astrogeology/plio/archive/refs/tags/v1.5.0.zip",
```

Comment:
Can releases point to github? Or do we need to re-release on gitlab?

Reply:
It should point to GitLab - having said that, we do not have any releases on there. Let's keep aimed at GitHub for now.